content/vault/v1.16.x/content/api-docs/secret/ssh.mdx

```diff
@@ -558,8 +558,12 @@ $ curl \
 ## Submit CA information
 
 This endpoint allows submitting the CA information for the secrets engine via an SSH
-key pair. _If you have already set a certificate and key, they will be
-overridden._
+key pair. 
+
+<Note>
+If a certificate or key are present, an error will be returned. To update the existing
+values, the Delete CA information API will need to be called first.
+</Note>
```

Comment:
```suggestion
```
There's no need to make this a standalone note. We can just fold it into the intro sentence describing the endpoint